torchtitan/experiments/flux/dataset/flux_dataset.py

```diff
@@ -233,11 +232,9 @@ def __iter__(self):
 
     def load_state_dict(self, state_dict):
         self._sample_idx = state_dict["sample_idx"]
-        self._all_samples = state_dict["all_samples"]
 
     def state_dict(self):
         return {
-            "all_samples": self._all_samples,
```

Comment:
Why remove this?

Reply:
Because we only do padding instead of packing for flux dataloader, so we don't need to save this field